### PR TITLE
New XCM-SDK Links

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

The owner of the XCM repo has changed so I've updated those links, plus a couple quick other things I noticed. 
CN PR: https://github.com/moonbeam-foundation/moonbeam-docs-cn/pull/401

### Checklist

- [x] I have added a label to this PR 🏷️
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Review by eshaben — COMMENTED** [review]
Nice! Thank you! Can you please fix the merge conflicts? 🙂

File: builders/build/historical-updates.md [hunk 1]
```diff
@@ -127,7 +127,8 @@ This bug existed during the following runtimes and block ranges:
 |   Moonriver    |    RT49    | RT2401 |     0 - 4668844      |
 | Moonbase Alpha |    RT40    | RT2401 |     0 - 4591616      |
 
-For more information, you can review the [relative PR on GitHub](https://github.com/PureStake/moonbeam/pull/2291){target=\_blank}.
+For more information, you can review the [relative PR on GitHub](https://github.com/moonbeam-foundation/moonbeam/pull/2291){target=_blank}.
+
```
Comment (eshaben) [inline builders/build/historical-updates.md:131]:
```suggestion
For more information, you can review the [relative PR on GitHub](https://github.com/moonbeam-foundation/moonbeam/pull/2291){target=\_blank}.
```

File: builders/interoperability/xcm/xcm-sdk/v0/xcm-sdk.md [hunk 2]
```diff
@@ -534,7 +534,7 @@ Where the returned values are as follows:
...
 |        `min`         |                                                                                                                                                            the minimum transferrable amount                                                                                                                                                             |
-|    `moonChainFee`    |                   the [asset](#assets) and amount that is needed to pay for Moonbeam's XCM fees. <br> If different from the `asset` to be transferred, the fee will be sent in this <br> asset in addition to the `asset` to be transferred (as of [v0.4.0](https://github.com/PureStake/xcm-sdk/releases/tag/v0.4.0){target=\_blank})                   |
+|    `moonChainFee`    |                   the [asset](#assets) and amount that is needed to pay for Moonbeam's XCM fees. <br> If different from the `asset` to be transferred, the fee will be sent in this <br> asset in addition to the `asset` to be transferred (as of [v0.4.0](https://github.com/moonbeam-foundation/xcm-sdk/releases/tag/v0.4.0){target=_blank})         |
```
Comment (eshaben) [inline builders/interoperability/xcm/xcm-sdk/v0/xcm-sdk.md:537]:
```suggestion
|    `moonChainFee`    |                   the [asset](#assets) and amount that is needed to pay for Moonbeam's XCM fees. <br> If different from the `asset` to be transferred, the fee will be sent in this <br> asset in addition to the `asset` to be transferred (as of [v0.4.0](https://github.com/moonbeam-foundation/xcm-sdk/releases/tag/v0.4.0){target=\_blank})         |
```

File: builders/interoperability/xcm/xcm-sdk/v1/xcm-sdk.md [hunk 3]
```diff
@@ -345,7 +345,8 @@ The process for using `assets` to build the transfer data is as follows:
 
     This will return the `accounts` function, which is used to define the source and destination addresses and the associated signers for each address
 
-The asset and chain objects are managed within the `@moonbeam-network/xcm-config` package. You do not need to directly interact with this package as the SDK exposes this data, but there you can find the list of [assets](https://github.com/PureStake/xcm-sdk/blob/main/packages/config/src/assets.ts){target=\_blank} and [chain data](https://github.com/PureStake/xcm-sdk/blob/main/packages/config/src/chains.ts){target=\_blank}.
+The asset and chain objects are managed within the `@moonbeam-network/xcm-config` package. You do not need to directly interact with this package as the SDK exposes this data, but there you can find the list of [assets](https://github.com/moonbeam-foundation/xcm-sdk/blob/main/packages/config/src/assets.ts){target=_blank} and [chain data](https://github.com/moonbeam-foundation/xcm-sdk/blob/main/packages/config/src/chains.ts){target=_blank}.
```
Comment (eshaben) [inline builders/interoperability/xcm/xcm-sdk/v1/xcm-sdk.md:348]:
```suggestion
The asset and chain objects are managed within the `@moonbeam-network/xcm-config` package. You do not need to directly interact with this package as the SDK exposes this data, but there you can find the list of [assets](https://github.com/moonbeam-foundation/xcm-sdk/blob/main/packages/config/src/assets.ts){target=_blank} and [chain data](https://github.com/moonbeam-foundation/xcm-sdk/blob/main/packages/config/src/chains.ts){target=\_blank}.
```

**Review by eshaben — COMMENTED** [review]
pushed a PR recently that changed target=_blank to be target=\_blank instead. so just gotta update these here. i'm afraid to commit these myself, because then i don't think i'll be able to merge it haha so can you please commit the suggestions? thank you!!

**Comment by themacexpert** [comment]
> pushed a PR recently that changed `target=_blank` to be `target=\_blank` instead. so just gotta update these here. i'm afraid to commit these myself, because then i don't think i'll be able to merge it haha so can you please commit the suggestions? thank you!!

Thank you!

**Review by eshaben — APPROVED** [review]
LGTM! ✨🚢

**Pull request merged**